src/classnames.ts

```diff
@@ -16,10 +18,21 @@ export type Source = {
   [key: string]: any;
 };
 
+/** @deprecated·replaced by $always */
 const $all = "$all";
+const $always = "$always";
 const $none = "$none";
 const $nil = "$nil";
 
+/**
+ * check if prop is directive $always
+ * @deprated must remove after completing deprecation of $all directive
+ */
+const isAlways = (prop: string): boolean => prop === $all || prop === $always;
+
+const getAlwaysVariation = (variations: Variations | string | null | undefined): MaybeString | Variations =>
+  variations[$all] || variations[$always];
```

Comment:
@jhesgodi shouldn't we have some kind of guard against `string`, `null` or `undefined` here ?! Or let me ask this way, do we expect `getAlwaysVariation` to be called with `string`, `null`, or `undefined` at all?